recipes/jupyterlab-transient-display-data/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "jupyterlab-transient-display-data" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://github.com/vatlab/transient-display-data/archive/{{ version }}.tar.gz
+  sha256: 3100e31e265a1ba169ec693058f4b00980176343c3090a959b139e865ab517a8
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - jupyterlab
+    - python >=3.5
+  run:
+    - jupyterlab
+    - nodejs
+    - python >=3.5
+
+test:
+  commands:
+    - jupyter labextension list
```

Comment:
I don't see how these commands are testing the package? Linux and macOS should be failing as they're empty due to the incorrect build script filename.

Reply:
Added `| grep transient-display-data` to test successful installation of the extension. The problem exists in other jupyterlab extension recipes though, for example, https://github.com/conda-forge/jupyterlab-plotly-extension-feedstock/blob/master/recipe/meta.yaml#L30, https://github.com/conda-forge/jupyterlab-git-feedstock/blob/master/recipe/meta.yaml#L34